### PR TITLE
Temporarily update deep strip

### PR DESCRIPTION
**Pull request by yurii-vasyliev**
Just a temporarily PR to get a preview URL.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement
 FROM node:18 AS yarn-dependencies
 WORKDIR /srv
 ADD package.json yarn.lock .
-RUN --mount=type=cache,target=/usr/local/share/.cache/yarn yarn install --production
+RUN --mount=type=cache,target=/usr/local/share/.cache/yarn PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1 yarn install --production
```
Comment (bartaz) [inline Dockerfile:17]: This is unrelated to ubuntu.com or this PR, I meant changing that on vanilla-framework.

**Comment by petesfrench** [comment]
@yurii-vasyliev Is this demo still needed? If not can we close it?

**Comment by yurii-vasyliev** [comment]
@petesfrench , nope, it's not needed. Thanks for checking!
